source/faq.txt

```diff
@@ -10,3 +10,71 @@ Frequently Asked Questions
 This page provides answers to some frequently asked questions we have
 encountered. If you have additional questions please contact MongoDB
 Support.
+
+Does ``mongosync`` have to use the same operating system as MongoDB Enterprise Server? 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+When ``mongosync`` runs on its own hardware, it does not have to use
+the same operation system (OS) as the source or destination clusters.
+
+Should I increase the size of the oplog in the source cluster?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you anticipate synchronizing a large data set, or if you plan to
+pause synchronization for an extended period of time, please increase
+the size of the replica set :term:`oplog` in the source cluster.
+
+The ``oplog`` in the source cluster must be large enough to track
+events that happen during the time it takes to complete the initial
+sync to the destination cluster.
+ 
+To learn more about how to increase the size of the ``oplog``, see:
+:ref:`tutorial-change-oplog-size`. 
+
+Which connection string options does ``mongosync`` allow?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+(e.g. connection string URI contains custom read preference, tags, etc)?
+
+``mongosync`` requires :ref:`readConcern: "majority" <read-concern>`
+and :ref:`writeConcern: "majority" <write-concern>`. ``mongosync`` 
+overwrites any other ``readConcern`` or ``writeConcern`` and uses
+``"majority"`` instead.
+
+``monogosync`` accepts all other :ref:`connection string options
+<mongodb-uri>`.
+
+
+Which security and authentication options are supported?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+``mongosync`` uses a standard MongoDB :ref:`connection string
+<mongodb-uri>` to connect to the source and destination clusters. 
+
+:ref:`LDAP <security-auth-ldap>` and :ref:`X509
+<authentication-mechanism-x509>` are supported. For available
+authentication options, see :ref:`authentication`.
+
+
+Can I configure ``mongosync`` for high availability?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+``mongosync`` is stateless. The metadata for restarting is stored on
+the destination cluster.
+
+A ``mongosync`` operation can be resumed if ``mongosync`` is
+unavailable during synchronization. When ``mongosync`` becomes
+available, restart the ``mongosync`` process with the same parameters.
+``mongosync`` resumes from where it left off.
+
+There is no automatic failover built into ``mongosync``, however you
+can write a script or use your operating system's process managers,
+``systemd`` for example, to restart the ``mongosync`` process. 
+
+
+Can the source or destination be a replica set with arbiters? 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Yes. The source replica-set must have more than 2 non-arbiter node.
+Please refer to read preference options to specify in the source cluster’s
```

Comment:
[comment]

I am having a hard time understanding what "refer to read preference options to specify in the source cluster's connection string" means. I'd consider rewriting.